### PR TITLE
adds epub validation for production + fixes discovered validation issues

### PR DESCRIPTION
**Pull request by mekarpeles**
Also:
- removes DAISY ACE (which wasn't implemented) EDIT: done in separate branch
- fixes duplicate authors
- fixes an introduced css typo
- fixes kdu_expand calls so they both use `-reduce` and `-no_seek`

### CODE REVIEW

File: abbyy_to_epub3/create_epub.py [hunk 1]
```diff
@@ -464,6 +465,9 @@ def make_chapter(self, heading, chapter_no):
         if not heading:
             heading = "Chapter {}".format(chapter_no)
 
+        # XXX
```
Comment (bfalling) [inline abbyy_to_epub3/create_epub.py]: SUGGESTION: Remove the triple X.

File: abbyy_to_epub3/create_epub.py [hunk 2]
```diff
@@ -1045,14 +1051,20 @@ def craft_epub(self, epub_outfile="out.epub", tmpdir=None):
                 'lr': 'ltr',
                 'rl': 'rtl',
             }
+
+            self.progression = "default"
```
Comment (bfalling) [inline abbyy_to_epub3/create_epub.py]: SUGGESTION: Single quotes for consistency?

File: abbyy_to_epub3/create_epub.py [hunk 3]
```diff
@@ -1123,12 +1135,22 @@ def craft_epub(self, epub_outfile="out.epub", tmpdir=None):
...
+        if not self.debug:
+            # we probably shouldn't hard code this list:
+            exceptions = ['Non-standard image resource of type image/x-ms-bmp found.']
+            errors = [err for err in result.messages if
+                      # filter out warnings (i.e. non errors) if not
+                      # in debug mode
+                      err.level.lower() == "error" and
+                      # filter our errors whose messages match known /
+                      # approved exceptions
+                      err.message not in exceptions]
```
Comment (bfalling) [inline abbyy_to_epub3/create_epub.py]: SUGGESTION: Maybe don't code so exactly to the entire line? e.g. just check for `image/x-ms-bmp found` present?

File: abbyy_to_epub3/create_epub.py [hunk 4]
```diff
@@ -1123,12 +1135,22 @@ def craft_epub(self, epub_outfile="out.epub", tmpdir=None):
...
-            if self.args.ace:
-                self.logger.info("Running DAISY Ace on {}".format(epub_outfile))
-                verifier.run_ace(epub_outfile)
+        # run validation on epub
+        if self.args and self.args.epubcheck:
+            self.validate_epub(epub_outfile)
+
+    def validate_epub(self, epub_file):
+        self.logger.debug("Running EpubCheck on {}".format(epub_file))
+        result = self.verifier.run_epubcheck(epub_file)
```
Comment (bfalling) [inline abbyy_to_epub3/create_epub.py]: QUESTION: Is `result` even used?

Comment (mekarpeles) [reply abbyy_to_epub3/create_epub.py]: yes -- `if not self.debug` uses `result.messages`

**Pull request merged**